src/Bootstrap/less/theme/page-display-package-v2.less

```
@@ -185,11 +185,56 @@
         margin-top: 3px;
       }
     }
+
+    .sidebar-headers {
+      font-size: 16px;
+      font-weight: 600;
+      color: @gray-light;
+      margin-bottom: 14px;
+    }
+
+    .sidebar-links {
+      font-size: 16px;
+    }
+
+    .title-links {
+      font-size: 14px;
+      font-weight: 400;
+      padding-top: 2px;
+
+      a {
+        @media (min-width: @screen-md-min) {
+          float: right;
```

It looks like Bootstrap's `.pull-right` adds an `!important`:

https://github.com/NuGet/NuGetGallery/blob/db62de43e24b704e9ab16d8abc39bdfa3e73260b/src/Bootstrap/less/utilities.less#L17-L19

I checked [Bootstrap's docs](https://getbootstrap.com/docs/3.3/css/#helper-classes-floats), and it turns out you can use a LESS mixin to effectively copy and paste the `.pull-right` styling here:

```suggestion
          .pull-right();
```

---

done 👍 the Bootstrap docs method worked!